packages/api/build.js

```diff
@@ -0,0 +1,22 @@
+const pkgJson = require('@npmcli/package-json')
+const { nodeExternalsPlugin } = require('esbuild-node-externals')
+
+const buildShallow =
+  process.argv.includes('--shallow') || process.env.ATP_BUILD_SHALLOW === 'true'
+
+if (process.argv.includes('--update-main-to-dist')) {
```

Comment:
this works for now. but we'll probably want a build + publish script that does this, publishes & then reverts. or some fancier copy files around stuff

Reply:
Totally agree.  For now this is little more than a stepping stone to unravel some things and enable the app to be instrumented.